build.gradle

```diff
@@ -23,12 +23,8 @@ task wrapper(type: Wrapper) {
 dependencies {
     // https://mvnrepository.com/artifact/org.bouncycastle/bcprov-jdk15on
     implementation group: 'org.bouncycastle', name: 'bcprov-jdk15on', version: '1.60'
```

Comment:
I need to use `org.bouncycastle:bcpkix-jdk15on` to satisfy all used classes in your code.